signer/src/storage/model.rs

```diff
@@ -1004,6 +1009,56 @@ impl PartialOrd for StacksPrincipal {
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
 pub struct ScriptPubKey(bitcoin::ScriptBuf);
 
+/// A taproot script hash.
+#[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub struct TaprootScriptHash(bitcoin::TapNodeHash);
+
+impl Deref for TaprootScriptHash {
+    type Target = bitcoin::TapNodeHash;
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl From<bitcoin::TapNodeHash> for TaprootScriptHash {
+    fn from(value: bitcoin::TapNodeHash) -> Self {
+        Self(value)
+    }
+}
+
+impl Default for TaprootScriptHash {
+    fn default() -> Self {
+        TaprootScriptHash::from_byte_array([0; 32])
+    }
+}
+
+impl TaprootScriptHash {
+    /// Create a new taproot script hash with all zeroes
+    pub fn new() -> Self {
+        Default::default()
+    }
```

Comment:
This should be feature gated for testing, since I do not think that we will use it in non-testing code.

Reply:
[ef319cf](https://github.com/stacks-sbtc/sbtc/pull/1703/commits/ef319cfb8efc40416cdaeafdcd25404d46b17cf9)

Reply:
Yeah good point. Or just rename this function to `zeros()` or something more specific.

Reply:
[8c314a7](https://github.com/stacks-sbtc/sbtc/pull/1703/commits/8c314a7267b23a5a61deedcd434ea77a35523b42)